xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-docker/pom.xml

```diff
@@ -90,6 +90,13 @@
       <version>${project.version}</version>
       <type>xar</type>
     </dependency>
+    <!-- Needed for the recycle bin skipping configuration -->
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-refactoring-ui</artifactId>
+      <version>${project.version}</version>
+      <type>xar</type>
```

Comment:
Shouldn't that be defined in the flamingo skin dependencies instead? (as a runtime dep). I don't see why it would be needed for the tests and not for XWiki at runtime.

Reply:
Added some documentation to explain why the dependency is currently required here https://github.com/xwiki/xwiki-platform/pull/1401/files#diff-b8f26bdf308131e60c13f82300ec6131R94